src/components/button/button.scss

```diff
@@ -230,15 +230,15 @@
 
 // button styles
 // solid
-:host {
+:host([appearance="solid"]) {
```

Comment:
To simplify our styles, let's not add selectors for default styles. No attribute here should mean `appearance=solid`.

This applies to any selectors explicitly using defaults.